pkg/apis/config/feature_flags.go

```diff
@@ -210,12 +203,13 @@ type FeatureFlags struct {
 	SetSecurityContextReadOnlyRootFilesystem bool   `json:"setSecurityContextReadOnlyRootFilesystem,omitempty"`
 	Coschedule                               string `json:"coschedule,omitempty"`
 	EnableCELInWhenExpression                bool   `json:"enableCELInWhenExpression,omitempty"`
-	EnableStepActions                        bool   `json:"enableStepActions,omitempty"`
 	EnableParamEnum                          bool   `json:"enableParamEnum,omitempty"`
 	EnableArtifacts                          bool   `json:"enableArtifacts,omitempty"`
 	DisableInlineSpec                        string `json:"disableInlineSpec,omitempty"`
 	EnableConciseResolverSyntax              bool   `json:"enableConciseResolverSyntax,omitempty"`
 	EnableKubernetesSidecar                  bool   `json:"enableKubernetesSidecar,omitempty"`
+	// EnableStepActions is a no-op flag since StepActions are stable
+	EnableStepActions bool `json:"enable-step-actions,omitempty"`
```

Comment:
NIT: text should be aligned line on the lines above

Reply:
fixed